server/views/applications/pages/risk-of-serious-harm/behaviour-notes-data.njk

```diff
@@ -0,0 +1,83 @@
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "../../../components/formFields/form-page-date-input/macro.njk" import formPageDateInput %}
+
+{% extends "../layout.njk" %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="{{ columnClasses | default("govuk-grid-column-three-quarters") }}">
+      <h1 class="govuk-heading-l govuk-!-margin-bottom-4">{{ page.title }}</h1>
+      <div class="govuk-body">
+        <p>Behaviour notes are positive or negative notes about an applicant's behaviour.</p>
+
+        <p>You should share behaviour notes from the last year, or over any period of time if they show a pattern of behaviour. Add one behaviour note at a time.</p>
+
+        <p>Examples of behaviour notes could be:</p>
+
+        <ul>
+          <li>issues with working or being managed by a specific gender of staff</li>
+          <li>inappropriate comments or behaviour</li>
+          <li>P-Nomis alerts</li>
+          <li>protective factors, such as positive relationships, employment and goals</li>
+        </ul>
+
+        {{ govukDetails({
+              summaryText: "What protective factors are",
+              text: "Protective factors can counteract negative influences. Individual protective factors include education, employment, leisure activities, goals and ambitions. Community and family protective factors include receiving professional support and having positive (prosocial) relationships."
+        }) }}
+      </div>
+      <form id="form" action="{{ paths.applications.appendToList({ id: applicationId, task: page.taskName, page: page.name }) }}?_method=PUT" method="post">
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+
+        {{ showErrorSummary(errorSummary) }}
```

Comment:
do we want to render the error summary here, or at the top of the page?

Reply:
yah good question, it should live inside the form - it's where it lives in the 'normal' form in `server/views/applications/pages/layout.njk` as well.